content/en/docs/devops-user-guide/how-to-use/jenkins-shared-library.md

```diff
@@ -0,0 +1,116 @@
+
+   - Under **Source Code Management**, choose **Git** and enter the URL of your repository of shared library for **Project Repository**. You have to configure **Credentials** if your repository requires one.
+
+     ![configure-shared-library](/images/docs/devops-user-guide/using-devops/jenkins-shared-library/configure-shared-library.png)
+
+5. When you finish editing, click **Apply**.
+
+{{< notice note >}}
+
+You can also specify [Folder-level Shared Libraries](https://www.jenkins.io/doc/book/pipeline/shared-libraries/#folder-level-shared-libraries).
+
+{{</ notice >}}
+
+## Use a Jenkins Shared Library in a KubeSphere DevOps Pipeline
+
+### Step 1: Create a pipeline
+
+1. Log in to the KubeSphere web console as `project-regular`. Go to your DevOps project and click **Create** on the **Pipelines** page.
+
+2. Set a name (for example, `demo-shared-library`) in the pop-up window and click **Next**.
+
+   ![set-name](/images/docs/devops-user-guide/using-devops/jenkins-shared-library/set-name.png)
+
+3. In **Advanced Settings**, click **Create** directly to create a pipeline with the default settings.
+
+   ![click-create](/images/docs/devops-user-guide/using-devops/jenkins-shared-library/click-create.png)
+
+### Step 2: Edit the pipeline
+
+1. In the pipeline list, click the pipeline to go to its detail page, and then click **Edit Jenkinsfile**.
+
+   ![edit-jenkinsfile](/images/docs/devops-user-guide/using-devops/jenkins-shared-library/edit-jenkinsfile.png)
+
+2. In the dialog that appears, enter the following example Jenkinsfile. When you finish editing, click **OK**.
+
+   ```groovy
+   library identifier: 'devops-ws-demo@master', retriever: modernSCM([
+       $class: 'GitSCMSource',
+       remote: 'https://github.com/devops-ws/jenkins-shared-library',
+       traits: [[$class: 'jenkins.plugins.git.traits.BranchDiscoveryTrait']]
+   ])
+   
+   //@Library('devops-ws-demo') _
+   //@Library('devops-ws-demo@test') _
+   
+   pipeline {
+       agent any
+   
+       stages {
+           stage('Demo') {
+               steps {
+                   script {
+                       mvn.fake()
+                   }
+               }
+           }
+       }
+   }
+   ```
+
+   {{< notice note >}}
+
+   You can specify a `label` for `agent` based on your needs.
+
+   {{</ notice >}}
+
+3. You can view the stage under the **Pipeline** tab. Click **Run** to run it.
+
+   ![click-run](/images/docs/devops-user-guide/using-devops/jenkins-shared-library/click-run.png)
+
+4. After a while, the pipeline will run successfully.
+
+   ![run-successfully](/images/docs/devops-user-guide/using-devops/jenkins-shared-library/run-successfully.png)
+
+5. You can click the **Success** record under **Status**, and then click **Show Logs** to view the log details.
+
+   ![log-details](/images/docs/devops-user-guide/using-devops/jenkins-shared-library/log-details.png)
+
+6. You can also use a Jenkinsfile starting with `@Library('demo-shared-library') _`  ???
```

Comment:
Is it necessary to demonstrate the method of using the shared library through a Jenkinsfile starting with `@Library('<the shared library name set in Jenkins>') _`?
As I understand it, this method is different from the method already demonstrated in the doc.

If so, I need someone to provide the Jenkinsfile...